### DISCUSSION TITLE
File path breadcrumbs are duplicated in code view

### DISCUSSION THREAD

**Discussion by jfrost-mo** [community/community#63779] [Repositories]
### Select Topic Area

Bug

### Body

See how the "example-python-project/src/Example" breadcrumbs is repeated/doubled up. This issue appears to occur over all repositories.

![image](https://github.com/community/community/assets/113985160/a80ab707-55a2-4029-939d-9c3032b1a7bc)

This occurs for me in Firefox on Linux (102.13.0esr). It does not on Firefox on Windows (116.0.2). That may be due to my Linux installation not using hardware acceleration.

I've tested in a private window, and the issue still occurs.
Being logged in or not does not make a difference.
The only extension I use is uBlock Origin, and I've tested without it, with the same result.

**Comment by whalesalad** [comment]
Surprised this is still happening. I am experiencing it, too. 
![Screenshot_20230829_164503](https://github.com/community/community/assets/5907/023bd504-5a1c-47c1-bc31-4a7534a3a817)

**Comment by TheDiamondfinderNG** [comment]
It's fixed now! Turns out it was because that version of firefox was outdated: https://github.com/orgs/community/discussions/61982#discussioncomment-6832854

**Comment by jfrost-mo** [comment]
As commented above this is due to FF 102 not supporting [container queries](https://developer.mozilla.org/en-US/docs/Web/CSS/@container). As FF 115.2 released on the 29th of August, FF 102 will be end of life by the end of November, and this will fix itself then. Most installations will auto update by the end of September.
